.circleci/config.yml

```
@@ -0,0 +1,19 @@
+version: 2.1
+
+jobs:
+  build-and-test:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - run:
+          name: Install Dependencies
+          command: npm ci
+      - run:
+          name: Run Tests
+          command: npm test
+
+workflows:
+  temp-workflow:
```

```suggestion
  ci:
```

If I understand what this field is about.

---

That's just a temporary name for now, since I don't know anything about the pipeline. I can changed it 'ci', however chances are we might have rename it again, once we add more stuff to it, or more workflows.